pkg/messages/messenger.go

```diff
@@ -0,0 +1,40 @@
+package messages
+
```

Comment:
Still need to implement a messaging system that can send messages up to SNS for MrRobot and Jambuildr